client.go

```diff
@@ -18,11 +18,11 @@ func getRedisConfig(cfgFile string) *RedisConfig {
 	return &config
 }
 
-func newRedisClient(cfgFile string) *redis.Client {
+func newRedisClient(cfgFile string, Addr string) *redis.Client {
 	config := getRedisConfig(cfgFile)
 	if config.ServerConfigs != nil {
 		return redis.NewClient(&redis.Options{
-			Addr:     config.ServerConfigs.Address,
+			Addr: Addr,
```

Comment:
use from cli if passed else fallback on config

Reply:
@darkLord19 isse better caller can decide what to pass. 
This function can then be taken to a new package

Reply:
works. newRedicClient ke input me can take model.RedisConfig as input